DataBiosphere/Saturn-UI
# [WM-2019] Method details in Submission Config page (in Workflows tab)

### Jira Ticket: https://broadworkbench.atlassian.net/browse/WM-2019




## Summary of changes:


### What
- Migration of the summary details portion of Submission Config page from CBAS UI

### Why
- Migrating CBAS UI into Terra UI

### Testing strategy
- Existing CBAS-UI test files copied over corresponding to the respective components/pages (only parts of SubmissionConfig.test.js were copied as the rest will be added in follow up tickets)

In browser:
- Enter Azure workspace + Launch Cromwell app
- Enable Workflows feature flag in console: configOverridesStore.set({ isAzureWorkflowsTabVisible: true })
- Refresh and click on Workflows tab
- Page should be populated with cards for 3 Covid-19 workflows
- Click on any card and it should load the Submission Config page with method details. Each tab in the page will be migrated in follow up tickets



https://github.com/DataBiosphere/terra-ui/assets/16748522/bcbded7d-4dd6-4935-b485-9ed015b81510
